deltachat-jsonrpc/src/api/types/calls.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::Result;
+
+use deltachat::context::Context;
+use deltachat::message::MsgId;
+use serde::Serialize;
+use typescript_type_def::TypeDef;
+
+#[derive(Serialize, TypeDef, schemars::JsonSchema)]
+#[serde(rename = "CallInfo", rename_all = "camelCase")]
+pub struct JsonrpcCallInfo {
+    /// True if the call is an incoming call.
+    pub is_incoming: bool,
+
+    /// True if the call should not ring anymore.
+    pub is_stale: bool,
+
+    /// True if the call is accepted.
+    pub is_accepted: bool,
+
+    /// True if the call has been ended.
+    pub is_ended: bool,
+
+    /// Call duration in seconds.
+    pub duration_seconds: i64,
```

Comment:
May be just `duration`, seconds are assumed by default, e.g. in `MessageObject` or `EphemeralTimer::Enabled`